dev/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# File generated by pre-commit: https://pre-commit.com
+# ID: 138fd403232d2ddd5efb44317e38bf03
+
+# start templated
+INSTALL_PYTHON=/usr/bin/python3
```

Comment:
do not use hardcoded absolute path for Python. Developers often relies on python env tools like venv and conda, which lead to relative python path.